Makefile

```diff
@@ -0,0 +1,65 @@
+ESLINT = node_modules/.bin/eslint
+NODEUNIT = node_modules/.bin/nodeunit
+XYZ = node_modules/.bin/xyz --repo git@github.com:fantasyland/fantasy-land.git --script scripts/prepublish
+
+BEL = $(shell printf '\007')
+ESC = $(shell printf '\033')
+
+
+.PHONY: all
+all: LICENSE index.d.ts index.js
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Fantasy Land/Copyright (c) $(shell git log --date=short --pretty=format:%ad | sort -r | head -n 1 | cut -d - -f 1) Fantasy Land/' '$@.orig' >'$@'
+	rm -- '$@.orig'
```

Comment:
This is an unrelated change which, in conjunction with __scripts/prepublish__, automatically keeps our copyright year up to date. I'm happy to open a separate pull request for this if desired.